instance/models/log_entry.py

```diff
@@ -21,24 +21,22 @@
 """
 
 # Imports #####################################################################
-
```

Comment:
Nit: We should keep this blank line to stay consistent with the formatting in other files.